spec/javascripts/required_field_spec.js

```diff
@@ -0,0 +1,48 @@
+describe("RequiredFields", function() {
+  var control = require('sufia/save_work/required_fields');
+
+  var title = '';
+  var selected = '';
+
+  describe("areComplete", function() {
+    var mockCheckbox = {
+      check: function() { },
+      uncheck: function() { },
+    };
+
+    describe("when required metadata is not present", function() {
+      beforeEach(function() {
+        var fixture = setFixtures(form('',''));
+        target = new control.RequiredFields(fixture.find('#new_generic_work'),function () {});
+      });
+      it("is not complete", function() {
+        expect(target.areComplete).toEqual(false);
+      });
+    });
+
+    describe("when required metadata is not present", function() {
+      beforeEach(function() {
+        var fixture = setFixtures(form('title','selected="selected"'));
+        target = new control.RequiredFields(fixture.find('#new_generic_work'),function () {});
+      });
+      it("is complete", function() {
+        expect(target.areComplete).toEqual(true);
+      });
+    });
+
+  });
+});
+
+function form(title, resource_type_selected) {
```

Comment:
have you considered using `loadFixtures('fixtures/form_with_single_value.html')`?